android/src/main/java/com/guichaguri/trackplayer/service/player/ExoPlayback.java

```diff
@@ -285,4 +286,8 @@ public void onPlaybackParametersChanged(PlaybackParameters playbackParameters) {
     public void onSeekProcessed() {
         // Finished seeking
     }
+   
+    public IcyMetadataListener onIcyMetadaUpdate(){
+        return manager.onIcyMetadaUpdate();
+    }
```

Comment:
Totally unnecessary method here, you can call the MusicManager directly instead of going through the player

Reply:
When I read the code, I ended up reading wrong and noticed that the `MusicManager manager` in `ExoPlayback.java` was set to `private`. But reading again I noticed that it is defined as 'protected', so the `LocalPlayback` class has access to it.

To 'delete' this function, I will have to change the scope of `toMediaSource`.

**From**:
```Java
public MediaSource toMediaSource (Context ctx, LocalPlayback playback) {
    ...
}
```
**To**:
```Java
public MediaSource toMediaSource (Context ctx, LocalPlayback playback, MusicManager manager) {
    ...
}
````

Changing also the `LocalPlayBack.java` file on lines 73 and 83 to:
> Linha 73:
```Java
source.addMediaSource(index, track.toMediaSource(context, this, manager), Utils.toRunnable(promise));
````
> Linha 83:
```Java
trackList.add(track.toMediaSource(context, this, manager));
````